README.md

```diff
@@ -296,6 +296,28 @@ interface. In case of a missing `interface_name`, the `name` of the profile name
 **Note:** The `name` (the profile name) and the `interface_name` (the device name) may be
 different or the profile may not be tied to an interface at all.
 
+### `interface_path`
+`interface_path` is a list of globbing pathnames(shell wildcard pattern) to match against the
+ID_PATH udev property of devices. udev property ID_PATH represents the persistent path of
+a device. It consists of a subsystem string(pci, usb, platform, etc.) and a subsystem-specific
+identifier. The subsystem-specific identifier for PCI device has format "pci-$domain:$bus:$device.$function",
+where each variable is an hexadecimal value. The path of a device can be obtained with "udevadm
+info /sys/class/net/$dev | grep ID_PATH=" or by looking at the "path" property exported by NetworkManager
+("nmcli -f general.path device show $dev"). The `interface_path` is optional and restricts the profile
+to be usable only on devices with the given `interface_path`. `interface_path` is only allowed for type
+ethernet or infiniband to match a non-virtual device with the profile.
+
+**Wildcard pattern matching table for `interface_path`:**
+| Pattern       | Meaning                           |
+| ------------- | ----------------------------------|
+| *             | matches everything                |
+| ?             | matches any single character      |
+| [foo]         | matches any character in foo      |
+| \|            | the element is optional, the match evaluates to be true if at least one of the optional element matches (logical OR). By default, an element is optional. This means that an element "foo" behaves the same as "\|foo"        |
```

Comment:
Is it possible to use ``` `|` ``` to make this less confusing in the plain text view? Maybe a definition list is nicer then a table, what do you think?

Reply:
Not possible to use ``` `|` ``` in  a table. Obviously, definition list will have few vertical bars and appear to be cleaner.